src/meetings/components/sidebar/visualEffectsAccordion/VisualEffectsList.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { FC, useMemo } from 'react';
+
+import { Container, Icon, Text } from '@zextras/carbonio-design-system';
+import { map } from 'lodash';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import useVirtualBackground from '../../../../hooks/useVirtualBackground';
+import { getBackgroundImage } from '../../../../store/selectors/ActiveMeetingSelectors';
+import useStore from '../../../../store/Store';
+import { VirtualBackgroundType } from '../../../../types/store/ActiveMeetingTypes';
+
+type VisualEffectCardsProps = {
+	meetingId: string;
+};
+
+const PictureContainer = styled(Container)<{ $picture?: string | false; $isSelected: boolean }>`
+	background-size: cover;
+	background-position: center;
+	aspect-ratio: 1.2959;
+	border-radius: 0.5rem;
+	${({ $isSelected, theme }): string | false =>
```

Comment:
https://github.com/user-attachments/assets/ab891e4d-bf06-4ee7-a053-8c2a98734d2c

To avoid this zoom effect, could you try with `outline` css property instead of `border`? Like tile effect

P.S.:  In your and UI opinion, wouldn't it be better to increase the border? I don't see it much